tests/phpunit/integration/ContextTest.php

```diff
@@ -191,7 +191,8 @@ public function test_get_reference_entity__in_admin_context() {
 		set_current_screen( 'toplevel_page_googlesitekit-dashboard' );
 		$_GET['page'] = 'googlesitekit-dashboard';
 		$this->assertNull( $context->get_reference_entity() );
-		$_GET['permaLink'] = home_url( '/non-existent-post/' );
+		// We can probably safely assume that 987654 is not a valid post ID in the test environment.
```

Comment:
A fairly safe assumption, but rather than assume, we can always add a quick assertion to make 💯  sure this is the case 😄 
```suggestion
		// We can probably safely assume that 987654 is not a valid post ID in the test environment, but let's make sure.
		$this->assertFalse( WP_Post::get_instance( 987654 ) );
```

Reply:
👍 😂